frontend/src/scenes/project-homepage/PrimaryDashboardModal.scss

```diff
@@ -0,0 +1,40 @@
+@import '~/vars';
+
+.primary-dashboard-modal {
```

Comment:
I feel like this could be handled with some existing (or new?) list component, instead of getting a custom one-off set of CSS styles. Building a `LemonList` comes to mind as a possible solution, but that might be too much for this 🤔 @Twixes ?

Reply:
100% agree. I started with `LemonTable`, but hit issues with spacing between rows + rounded corners. Had it kind of working, but it was such a hack that I re-did it like this last night.

I think it makes sense to break this out into a new component. We use something similar here and in the events table in recordings - maybe elsewhere too?

Reply:
So I started down the path of making `LemonList`, and came to the conclusion we need some more examples of what the `LemonList` should be. Looking at the overlap in style between this list + recordings, it ended up being that they both iterate data + have default_padding....

With that said, refactoring this list into being  a list of `LemonButton`s and `LemonRow`s got rid of a good bit of custom styles. So I did that.